src/daemon/screenshot.vala

```diff
@@ -372,6 +387,9 @@ namespace BudgieScr {
 			// show pointer
 			windowstate.screenshot_settings.bind("include-cursor", showpointerswitch, "active",	SettingsBindFlags.GET|SettingsBindFlags.SET);
 
+			// screenshot capture sound
+			windowstate.screenshot_settings.bind("screenshot-capture-sound", screenshotcapturesoundswitch, "active",	SettingsBindFlags.GET|SettingsBindFlags.SET);
```

Comment:
Github shows a tab before the flags. Not sure if that's real, or what's going on there.